packages/sdk-react/src/mocks/counterparts/address/counterpartsAddressesHandlers.ts

```diff
@@ -1,24 +1,19 @@
-import {
-  ADDRESSES_ENDPOINT,
-  CounterpartAddress,
-  CounterpartAddressResourceList,
-  CounterpartAddressResponseWithCounterpartID,
-  ErrorSchemaResponse,
-} from '@monite/sdk-api';
+import { components } from '@/api';
 
 import { http, HttpResponse, delay } from 'msw';
 
 import { counterpartsAddressesFixture } from './counterpartsAddressesFixture';
 
-const counterpartsAddressesPath = `*/counterparts/:counterpartId/${ADDRESSES_ENDPOINT}`;
+const counterpartsAddressesPath = `*/counterparts/:counterpartId/addresses`;
```

Comment:
Not even a recommendation, I just wanted to share how I do it in Onboarding:
```ts
const personsPath: `*${Extract<keyof paths, '/persons'>}` = `*/persons`;
```

Reply:
Ouch, MSW does not support OpenAPI syntax such a `{counterpart_id}`, it expects `:counterpart_id`. So, forget, and never mind about that 😶‍🌫️